Source/art/ftfont.m

```diff
@@ -56,7 +45,7 @@
 
 #import "ftfont.h"
 #import "FTFontEnumerator.h"
-#import "FTFaceInfo.h"
+// #import "FTFaceInfo.h"
```

Comment:
Please remove this now commented out line.

Reply:
Done.